src/using-document-creator.mdx

```diff
@@ -0,0 +1,179 @@
+
+To use this functionality, you will need a configuration file(`.json` file) and some pre-requisite informations.
+
+## What is document creator about?
+
+It's to provide a UI to the staffs that are involve with the daily operations who do not need to know anything about blockchain.
+
+## Why do we do this?
+
+Mainly because we want to cater to small firms or companies that doesn't really have the capabilities to build deep integration with their existing softwares or workflows.
+
+## How is this document creator helping small firms and companies?
+
+One way is for the IT department or someone who is tech savvy enough, can set up the configuration file and clerks or staffs can use this document creator to create the documents.
+
+## Pre-requisite
+
+Before you construct your configuration file(`.json` file), please ensure you have these:
+
+- [Etherum Wallet](./deploying-store)
+- [Document Store (for verifiable document)](./deploying-store)
+- [Token Registry (for transferable document)](./token-registry)
+- [Configured DNS](/dns-txt#how-to-create-dns-txt-record)
+- [Custom Renderer](./custom-template)
+
+Please refer to the approprate links to prepare them.
+
+## Configuration file
+
+This configuration file is a JSON file that contains information to configure the application to suit your business needs.
+
+### Config file structure
+
+```json
+{
+  "network": "ropsten",
+  "wallet": "{\"address\":\"6a36c563a5350d7be66c801f901a67e413adbdfa\",\"id\":\"9e2d12d0-fb5b-49d4-9aa4-a0eb5b665233\",\"version\":3,\"Crypto\":{\"cipher\":\"aes-128-ctr\",\"cipherparams\":{\"iv\":\"127c21b337a16bc24621e24e73614378\"},\"ciphertext\":\"a78db406835261f59394397b2682088eb2ada80552908bd5cfd92c915def4c54\",\"kdf\":\"scrypt\",\"kdfparams\":{\"salt\":\"8bb0ac10564a4297c6f934fd6a08c25cd3f7958b31dd13eb6e0e30eebd8fcaba\",\"n\":131072,\"dklen\":32,\"p\":1,\"r\":8},\"mac\":\"1b1821d6f4186ce34952196142602ffacffff6f3a5a603c7cd02d301c47ab697\"},\"x-ethers\":{\"client\":\"ethers.js\",\"gethFilename\":\"UTC--2020-07-24T03-18-04.0Z--6a36c563a5350d7be66c801f901a67e413adbdfa\",\"mnemonicCounter\":\"8cf993a436e09d6f06d81dec993c6fec\",\"mnemonicCiphertext\":\"56a1a1d07a1d856514fc7fb03c553a77\",\"path\":\"m/44'/60'/0'/0/0\",\"version\":\"0.1\"}}",
+  "forms": [{...}]
+}
+```
+
+### `"network"` field
+
+The `network` field is a string that refers to the network in which the document is created in.
+
+### `"wallet"` field
+
+The `wallet` field is a string that refers to your etherum wallet.
+
+To convert the wallet to a string like the above use this [online tool](https://tools.knowledgewalls.com/jsontostring).
+
+### Forms structure
+
+The `forms` field is an array of form object. The required fields for each form is `name`, `type`, `defaults` and `schema`. Other field such as `attachments` is also recognised.
+
+```json
+{
+  "name": "Bill of Lading",
+  "type": "TRANSFERABLE_RECORD",
+  "defaults": {...},
+  "schema": {...},
+  "attachments": {...}
```

Comment:
it's not an array ?

Reply:
it's not an array indeed, but plural is weird imho but not related to the doc

Reply:
ok, if there is a change then I'll update the doc accordingly.